game_eggs/steamcmd_servers/the_isle/evrima/egg-the-isle.json

```diff
@@ -48,9 +48,39 @@
             "rules": "required|numeric|digits_between:1,100",
             "field_type": "text"
         },
+        {
+            "name": "Allow Replay",
+            "description": "",
+            "env_variable": "ALLOW_REPLAY",
+            "default_value": "false",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|in:true,false",
+            "field_type": "text"
+        },
+        {
+            "name": "Humans",
+            "description": "Enable or disable humans. true or false",
+            "env_variable": "HUMANS",
+            "default_value": "false",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|in:true,false",
+            "field_type": "text"
+        },
+        {
+            "name": "Enable Dynamic weather",
+            "description": "",
+            "env_variable": "DYNAMIC_WEATHER",
+            "default_value": "true",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|in:true,false",
+            "field_type": "text"
+        },
         {
             "name": "RCON",
-            "description": "Enable RCON",
+            "description": "true or false",
```

Comment:
Leave this at `Enable RCON`

but set the rules to `required|string|in:true,false`